tools/spelling/spelling_dictionary.txt

```diff
@@ -401,6 +401,7 @@ boolean
 booleans
 bools
 borks
+br
```

Comment:
Sorry but to avoid adding this, can you potentially try adding backticks around the thing that triggered this? I think this is what @zuercher did in the associated PR. @zuercher perhaps we should have some error text that guides people on different ways of fixing the issue so we avoid more things going in the dictionary that shouldn't?

Reply:
Sure, I removed `br` completely.